src/__tests__/randomThings.test.js

```diff
@@ -0,0 +1,71 @@
+import * as rt from '../app/randomThings'
+
+describe('random number generation...', () => {
+  it('returns random int within requested (inclusive) limits', () => {
+    const randomInteger = rt.randomInteger(0, 1)
+
+    expect(Number.isInteger(randomInteger)).toBe(true)
+
+    expect(randomInteger).toBeGreaterThanOrEqual(0)
+    expect(randomInteger).toBeLessThanOrEqual(1)
+
+    expect(rt.randomInteger(0, 0)).toEqual(0)
+  })
+
+  it('errors when requesting random int with invalid params', () => {
+    expect(() => { rt.randomInteger(0) }).toThrow()
+    expect(() => { rt.randomInteger(1, 0) }).toThrow()
+    expect(() => { rt.randomInteger(':)', 0) }).toThrow()
+  })
+
+  it('returns sorted array of two random ints within req (incl) limits', () => {
+    const randomRange = rt.randomRange(0, 9)
+
+    expect(randomRange instanceof Array).toBe(true)
+
+    expect(randomRange[0]).toBeGreaterThanOrEqual(0)
+    expect(randomRange[0]).toBeLessThanOrEqual(9)
+    expect(randomRange[1]).toBeGreaterThanOrEqual(0)
+    expect(randomRange[1]).toBeLessThanOrEqual(9)
+    expect(randomRange[0]).toBeLessThan(randomRange[1])
+
+    expect(rt.randomRange(0, 1)).toEqual([0, 1])
+  })
+
+  it('errors when requesting sorted random ints with invalid params', () => {
+    expect(() => { rt.randomRange(0) }).toThrow()
```

Comment:
you can dry this out with an array of invalid params. makes it easier in the long run to add new variations as you think of them.